File: dataset/output/GEMINI_gemini-1.5-pro-001/Task1_GEMINI_gemini-1.5-pro-001.cpp
#include <vector>

int number(const std::vector<std::pair<int, int>>& bus_stops) {
  int people_on_bus = 0;
  for (const auto& stop : bus_stops) {
    people_on_bus += stop.first - stop.second;
  }
  return people_on_bus;
}

File: dataset/output/GEMINI_gemini-1.5-pro-001/task1/Task1_GEMINI_gemini_1_5_pro_001.java
public class Task1_GEMINI_gemini_1_5_pro_001 {
    public static int number(int[][] bus_stops) {
        int peopleOnBus = 0;
        for (int[] stop : bus_stops) {
            peopleOnBus += stop[0] - stop[1];
        }
        return peopleOnBus;
    }
}